docs/l4-l7.rst

```diff
@@ -105,13 +107,15 @@ Create an application using Advanced mode (Application subtab)
       Application stub code is displayed in the text box.
 4. In the :guilabel:`Application Name` field, enter a valid application name.
 5. Update the stub code which appears in the JSON box with appropriate values.
-6. Optionally, Click on :guilabel:`Manage Endpoint Mappings` to add Dynamic Endpoint Mappings for endpoint discovery. You can select APIC Tenant, Application, Endpoint Group from the provided drop-downs to associate them with the Application’s pool. Whenever the APIC Endpoint Group will have an endpoint list updated, the AS3 application’s pool will also have the same set of endpoints in it on the BIG-IP. 
+6. Optionally, Click on :guilabel:`Manage Endpoint Mappings` to add Dynamic Endpoint Mappings for endpoint discovery. You can select APIC Tenant, Application, Endpoint Group from the provided drop-downs to associate them with the Application’s pool. The VRF field is a read-only field and is retrieved from the APIC’s Endpoint Group itself. You can select Route Domain from the dropdown options to associate them with the Application's pool. Whenever the APIC Endpoint Group will have an endpoint list updated, the AS3 application’s pool will also have the same set of endpoints in it on the BIG-IP. 
 7. Click :guilabel:`Submit`.
 
 The new application is created on this BIG-IP device in the specified partition, and the app is listed in the Application list.
 
 .. note::
 
+   - Route Domain must be created on the BIG-IP since the L2-L3 Network Management Tab as well as AS3 configuration does not support creation of Route Domains.
```

Comment:
Either "Route Domains..." or "The Route Domain..."
... the L2-L3 Network Management tab, as well as the AS3 configuration, does not....